code/modules/mob/mob.dm

```diff
@@ -513,7 +513,12 @@
 		return
 	if(isAI(M))
 		return
-	show_inv(usr)
+
+/mob/MouseDrop_T(atom/dropping, atom/user)
```

Comment:
so src is the mob doing the mousedrop or is user?

wtfffff

Reply:
I don't even remember the mousedrop shit doesn't work in a sane manner and I was confused myself by that this works.

Reply:
I figured this out by some `to_chat()` prints of what each parameter was and it was pretty consistently unexpected from what I remember. But again, this was the working solution.

Reply:
yikes lol

Reply:
yikes x2, thanks for the info tho